Lib/test/support/__init__.py

```diff
@@ -2118,3 +2119,19 @@ def clear_ignored_deprecations(*tokens: object) -> None:
     if warnings.filters != new_filters:
         warnings.filters[:] = new_filters
         warnings._filters_mutated()
+
+
+# This number should be CPython internal "hot" count times 2.
+CPYTHON_WARMUP_COUNT = 8*2
+
+
+def cpython_warmup(f):
```

Comment:
It's not just a warmup, it does actually run a function n times. I suggest the name:

```suggestion
def repeat_cpython_adaptative(f):
```